date: 2025-04-22  
time: 18:24  

  - Exception define scope with in program, when your program execute in the try portion of a try-catch-finally statement, you are stating that execution can abort at any point and then resume at the catch.
  - But we can use try with resource but if class implement auto close-able.
  - And We have also global exception handler that handler error so we don't need to put the try catch every where.

Day 1 : done *2025-04-22*  
Day 3 : done *2025-04-25*  
Day 7 : done *2025-04-29*  
Day 21: done *2025-05-13*